tests/test-notebook/src/notebooktools.spec.ts

```diff
@@ -125,13 +128,16 @@ describe('@jupyterlab/notebook', () => {
       tabpanel.addWidget(notebookTools);
       tabpanel.node.style.height = '800px';
       Widget.attach(tabpanel, document.body);
-      // Give the posted messages a chance to be handled.
-      await sleep();
```

Comment:
Is there a reason to delete this? Seems like it could be useful for handling `onAfterAttach`, etc, as the comment suggests.

Reply:
My goal is definitely to clean up resource allocation/destruction in the suite to the point where such kludges aren't necessary. But I guess there's no way to explicitly await the messages (`BeforeAttach`, `AfterAttach`, etc?) that get sent as a result of the `addWidget` and `attach` calls? In that case I'll add it back.

Reply:
I agree with your goal here -- but in this case I don't think there is an explicit way to await handling these messages.

Reply:
That's too bad. I put it back